b2share/modules/schemas/views.py

```diff
@@ -139,12 +147,79 @@ def get(self, community_schema):
         return community_schema
 
 
+class BlockSchemaResource(ContentNegotiatedMethodView):
+    view_name = 'block_schema_creating_item'
```

Comment:
We should follow the endpoint naming of the other Invenio modules. Here it would be `block_schema_list `